Formula/alertmanager.rb

```diff
@@ -0,0 +1,88 @@
+class Alertmanager < Formula
+  desc "Prometheus Alertmanager"
+  homepage "https://prometheus.io/docs/alerting/alertmanager/"
+  url "https://github.com/prometheus/alertmanager/archive/v0.20.0.tar.gz"
+  sha256 "4789ef95b09ba86a66a2923c3535d1bfe30a566390770784c52052c7c83ee1bc"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/prometheus/alertmanager"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/alertmanager"
+
+    system "make", "build"
+    bin.install %w[alertmanager amtool]
+  end
+
+  def post_install
+    (var/"alertmanager").mkdir
+
+    (etc/"alertmanager.sh").write <<~EOS
```

Comment:
No, that's not going to be acceptable. If upstream wants to have a shell script with default options, they should ship it. But we are not maintaining inside the formula a list of non-default options.